provider/k8s.go

```diff
@@ -633,6 +637,37 @@ func (store genericFileStore) ServeDirectory(dir string) (Iterator, error) {
 	return parquetIteratorOverMultipleFiles(fileParts, store)
 }
 
+func (store genericFileStore) Upload(sourcePath string, destPath string) error {
+	content, err := ioutil.ReadFile(sourcePath)
+	if err != nil {
+		return fmt.Errorf("cannot read %s file because %v", sourcePath, err)
```

Comment:
typically the convention in go is to have <outer error>: <inner error>
`Errorf("cannot read file %s: %v"), sourcePath, err`